### DISCUSSION TITLE
How to specify which struct to use if you have several with the same name?

### DISCUSSION THREAD

**Discussion by diegolatorre116** [swaggo/swag#1807] [Q&A]
I am using the swaggo/swag library to generate the documentation automatically in my Gin application. So, I have the following endpoint for example that, as you can see, returns models.HealthCheck, which is a struct:

```
package healthcheck

import (
	"net/http"

	"github.com/gin-gonic/gin"
	"github.com/guanes/auth-terrai/app/domain/services"
)

// Health Check
// @Summary
// @Schemes
// @Tags Health Check
// @Accept json
// @Produce json
// @Success 200 {object} models.HealthCheck
// @Router /api/health-check [get]
func getHealthCheck(c *gin.Context) {
	healthCheck := services.GetHealthCheck()
	c.JSON(http.StatusOK, healthCheck)
}
```

The problem is that in my application I have two packages called models and both have the struct called HealthCheck, so I get this error:
 ParseComment error in file /go/src/gitlab.com/main/server/app/api/handlers/healthcheck/get.go :cannot find type definition: models.HealthCheck
Besides the obvious solution of calling different structs... is there a way to tell swag which one to use? I tried aliasing it when importing but it doesn't work either.

**Comment by sdghchj** [comment]
```
package healthcheck

import (
	"net/http"

	"github.com/gin-gonic/gin"
	"github.com/guanes/auth-terrai/app/domain/services"
        modelsA  "your/path/to/your/models"
)

// Health Check
// @Summary
// @Schemes
// @Tags Health Check
// @Accept json
// @Produce json
// @Success 200 {object} modelsA.HealthCheck
// @Router /api/health-check [get]
func getHealthCheck(c *gin.Context) {
	healthCheck := services.GetHealthCheck()
	c.JSON(http.StatusOK, healthCheck)
}
```
